### ISSUE TITLE
Gmail issue 

### ISSUE THREAD

**Issue by dkalyangoud** [mailcow/mailcow-dockerized#2400]
Describe the bug, try to make it reproducible:
Mails not being sent to Gmail accounts.
All the mails sent to any Gmail recipient are held up in the Queue manager, with Queue status - 'Deferred', and error - (delivery temporarily suspended: lost connection with alt4.gmail-smtp-in.l.google.com[64.233.177.26] while receiving the initial server greeting)

**Comment by andryyy** [comment]
No info?

I mean, at least post the queue manager logs.

Check the issue template, don’t remove the content but read it if you need help. :( Run the commands from the issue template and post the results.

**Comment by dkalyangoud** [comment]
> No info?
> 
> I mean, at least post the queue manager logs.
> 
> Check the issue template, don’t remove the content but read it if you need help. :( Run the commands from the issue template and post the results.

Sorry I didn't find the template while accessing through mobile.
Anyway, have now updated the post with logs.

**Comment by dkalyangoud** [comment]
Any help with the issue would be really appreciated

**Comment by speedmann** [comment]
Please remove the log as you are publishing several mail addresses which should not be public.

Where is your server hosted? 
Can you do a `nc -vz alt4.gmail-smtp-in.l.google.com 25` and a `traceroute alt4.gmail-smtp-in.l.google.com`
It might be the outgoing connection to port 25 is blocked by your firewall/ISP

**Comment by hachre** [comment]
My best guess is your server IP is blacklisted, try the check at:
https://mxtoolbox.com/blacklists.aspx

It could also be some kind of outgoing firewall in your server's network or running on your server directly.

Alternative typical reasons would be no MX, incorrect MX, completely inexistent domain name set up as your Mailcow host name and so on.

**Comment by dkalyangoud** [comment]
Thank you guys...the issue seems to be with the ISP; it's resolved now!

**Comment by andryyy** [comment]
Nice
